tests/runtime/data/stackdriver/stackdriver_test_timestamp.h

```diff
@@ -0,0 +1,62 @@
+/* 2020-07-21T16:40:42.000012345Z */
```

Comment:
Is this for `1591111124`, or for the timestamp object? Please make it clear.

Reply:
Yes.
Comments updated.